### PR TITLE
EOS-23280: Factory reset(data+logs) command implementation failing as After cluster start Utils Kafka services were in failed state

### PR DESCRIPTION
**Pull request by vastradparayya**
Signed-off-by: Parayya Vastrad <parayya.vastrad@seagate.com>

# Problem Statement
- Factory reset(data+logs) command implementation failing as After cluster start Utils Kafka services were in failed state
- BUG

# Design
-  https://jts.seagate.com/browse/EOS-23280

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: NA
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: N
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -258,32 +258,27 @@ def test(plan: str):
     @staticmethod
     def reset():
         """ Remove/Delete all the data that was created after post install """
```
Comment (sachinpunadikar) [inline py-utils/src/setup/utils.py:260]: Change this comment. 
""" Remove/Delete all the data/logs that was created by user/testing """

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
work on the comment

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/setup/utils.py [hunk 2]
```diff
@@ -257,33 +257,28 @@ def test(plan: str):
...
     @staticmethod
     def reset():
-        """ Remove/Delete all the data that was created after post install """
+        """ Remove/Delete all the data/logs that was created by user/testing """
+        # TODO
+        return 0
+
+    @staticmethod
+    def cleanup():
+        """Remove/Delete all the data that was created after post install."""
```
Comment (selvakumaar5496) [inline py-utils/src/setup/utils.py:266]: Please be consistent on comments

**Review by selvakumaar5496 — APPROVED** [review]

File: py-utils/src/setup/utils.py (hunk 1 above)
Comment (vastradparayya) [reply py-utils/src/setup/utils.py]: Done

**Pull request merged**